### PR TITLE
blockdev_stream_to_invalid_node:support new error msg in qemu6.1

### PR DESCRIPTION
**Pull request by aliang123**
Error msg is changed after qemu6.1, add support for it

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:2004327

### CODE REVIEW

**Comment by aliang123** [comment]
qemu6.1:
   virtio_scsi: (1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_stream_to_invalid_node.q35: PASS (42.49 s)
   virtio_blk:(1/1) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_stream_to_invalid_node.q35: PASS (43.16 s)

qemu6.0:
  virtio_scsi:
    (1/1) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_stream_to_invalid_node.q35: PASS (41.52 s)
  virtio_blk:
   (1/1) Host_RHEL.m8.u5.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_stream_to_invalid_node.q35: PASS (41.69 s)

**Comment by aliang123** [comment]
@zhencliu @ngu-niny, please help to review, thanks.

File: qemu/tests/cfg/blockdev_stream_to_invalid_node.cfg [hunk 1]
```diff
@@ -25,8 +25,11 @@
     rebase_mode = unsafe
     mount_point = "/var/tmp"
     qemu_force_use_drive_expression = no
+    required_qemu_version = [6.1.0, )
 
     virtio_scsi:
-        qmp_error_msg = "Conflicts with use by ${device_tag} as 'root', which uses 'write' on %s"
+        qmp_error_msg_6_0 = "Conflicts with use by ${device_tag} as 'root', which uses 'write' on %s"
+        qmp_error_msg_6_1 = "permissions 'write' are both required by block device '${device_tag}' (uses node '%s' as 'root' child)"
```
Comment (zhencliu) [inline qemu/tests/cfg/blockdev_stream_to_invalid_node.cfg:32]: what about qmp_error_before_6_1 qmp_error_after_6_1, for err msg is changed since qemu 6.1

**Comment by zhencliu** [comment]
LGTM

**Comment by aliang123** [comment]
qemu6.1:
   virtio_blk: (1/1) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_stream_to_invalid_node.q35: PASS (101.43 s)
   virtio_scsi:(1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_stream_to_invalid_node.q35: PASS (43.31 s)

qemu6.0:
   virtio_blk:(1/1) Host_RHEL.m8.u5.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_stream_to_invalid_node.q35: PASS (37.59 s)
  virtio_scsi:(1/1) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_stream_to_invalid_node.q35: PASS (39.90 s)

**Comment by ngu-niny** [comment]
(1/1) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_to_invalid_node: PASS (109.57 s)
 (2/2) Host_RHEL.m8.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.ppc64le.io-github-autotest-qemu.blockdev_stream_to_invalid_node: PASS (115.64 s)

Acked-by: Nini Gu <ngu@redhat.com>

**Pull request merged**